src/server.py

```diff
@@ -0,0 +1,34 @@
+"""Something server-y."""
+
+import socket
+
+
+def server():
+    """Start a server at localhost:9999."""
+    server = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
+    address = ('127.0.0.1', 5001)
+    server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+
+    server.bind(address)
+
+    server.listen(1)
+
+    while True:
+
+        conn, addr = server.accept()
+
+        try:
+            message = ""
```

Comment:
If you're using this to accumulate a message from the client then it needs to be a bytestring no matter what version of Python you're running. This breaks in Python 3.